runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -80,7 +80,7 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 	MMINLINE uintptr_t
 	contiguousHeaderSize()
 	{
-		return compressObjectReferences() ? sizeof(J9IndexableObjectContiguousCompressed) : sizeof(J9IndexableObjectContiguousFull);
+		return compressObjectReferences() ? static_cast<J9JavaVM*>(_omrVM->_language_vm)->sizeOfJ9IndexableObjectContiguousCompressed : static_cast<J9JavaVM*>(_omrVM->_language_vm)->sizeOfJ9IndexableObjectContiguousFull;
```

Comment:
I think this helper is called most often. I am not sure how much perf improvement we can get by caching header sizes in this class locally (and btw if cached these variables might be declared `const`)

Reply:
Yes, this class is a good candidate for caching those sizes.